niworkflows/anat/mni.py

```diff
@@ -164,35 +170,115 @@ def _config_ants(self, ants_settings):
 
     def _generate_report(self):
         """ Generates the visual report """
-        NIWORKFLOWS_LOG.debug('Generating visual report')
+        NIWORKFLOWS_LOG.info('Generating visual report')
 
-        def _plot_xyz(anat_img, plot_params, div_id):
+        def _plot_xyz(anat_img, div_id, plot_params=None,
+                      order=('z', 'x', 'y'), cuts=None):
             """
-
+            Plots the foreground and background views
+            Default order is: axial, coronal, sagittal
             """
-            NIWORKFLOWS_LOG.info('Plotting %s', str(anat_img))
-            svgs = []
+            if plot_params is None:
+                plot_params = {}
 
-            for mode, cuts in list(DEFAULT_CUTS.items()):
+            # Use default MNI cuts if none defined
+            if cuts is None:
+                cuts = DEFAULT_MNI_CUTS.copy()
+
+            NIWORKFLOWS_LOG.debug('Plotting %s for interface report.', anat_img)
+            out_files = []
+
+            # Plot each cut axis
+            for mode in list(order):
+                out_file = '{}_{}.svg'.format(div_id, mode)
                 plot_params['display_mode'] = mode
-                plot_params['cut_coords'] = cuts
-                image = plot_anat(anat_img, **plot_params)
-                svgs.append('<div id="{0}-{1}" class="{0}">{2}</div>'.format(
-                    div_id, mode, report.as_svg(image)))
-                image.close()
-            return '\n'.join(svgs)
-
-        plot_params = {'cut_coords': 3}
-        report.save_html(
```

Comment:
Because I don't need it. I think the visual reports should be as agnostic as possible about where they will end up included. With that in mind, a standalone svg is much better than decorating it with html code.

Example use case: if I want to directly import one artifact of this reports as a figure in a latex paper. If you don't produce svg you will not be able to do this directly: you'll need to remove html decorations.